README.md

```diff
@@ -2,51 +2,77 @@
 
 The [Scroll Behavior specification](https://developer.mozilla.org/en/docs/Web/CSS/scroll-behavior) has been introduced as an extension of the `Window` interface to allow for the developer to opt in to native smooth scrolling. To date this has only been implemented in _Firefox_.
 
-See it in action https://iamdustan.github.io/smoothscroll
+Check out all the methods covered here: https://iamdustan.github.io/smoothscroll
 
 
-## Install
+## Installation and use
 
-Download the distribution file from this repository and include it in your project.
+Download the production ready file [here](https://unpkg.com/smoothscroll-polyfill/dist/smoothscroll.min.js) and include it in your project, or install it as a package.
 
-You can also find it in **npm** as **smoothscroll-polyfill** or **bower** as **smoothscroll**.
+```sh
+# npm
+npm install smoothscroll-polyfill --save
+
+# yarn
+yarn add smoothscroll-polyfill
+```
+
+When including the polyfill in a script tag, it will run immediately after loaded. If you are importing it as a dependency, make sure to call the `polyfill` method:
 
 ```js
-require('smoothscroll-polyfill').polyfill();
+import smoothscroll from 'smoothscroll-polyfill';
+
+// kick off the polyfill!
+smoothscroll.polyfill();
 ```
 
-_Requires requestAnimationFrame polyfill for browsers which don't support it!_
+In both cases, the script will detect if the spec is natively supported and take action only when necessary.
+
+_The code requires requestAnimationFrame polyfill for browsers which don't support it._
 
 
 ### Force polyfill implementation
 
-If you prefer the polyfill to always override the current native methods (sometimes needed because of partial implementations or inconsistencies between browsers), place this global variable before requiring the module or including the polyfill file.
+If you prefer to always override the native scrolling methods, place this global variable before requiring the module or including the polyfill file.
 
 ```js
 window.__forceSmoothScrollPolyfill__ = true;
```

Comment:
I strongly think we should change this to `require('smoothscroll-polyfill').polyfill({ forceOverride: true})` or something equally verbose but entirely controllable via javascript and not some odd global flag

Reply:
Yeah, we would need to expose the `polyfill` method for script tag additions too, which I'm not against, but would be a major.

We can create an issue for that and I will take care of it, it's no-brainer.

Reply:
I thought of that while writing #108 😅 

Let’s do both so each usage has a first-class-ish solution.